charts/observability-pipelines-worker/templates/daemonset.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if (eq .Values.role "Agent") -}}
```

Comment:
@lukesteensen @binarylogic there was discussion about only supporting the service/aggregator models. If we can confirm that there's quite a bit of templating that can get cut away.

Reply:
Yes, it should be safe to remove anything related to the agent-style model. OP for the foreseeable future will focus on the aggregator use case.